paymentsheet/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -152,12 +166,14 @@ internal abstract class BaseAddPaymentMethodFragment : Fragment() {
                     },
                     formArguments = arguments,
                     onFormFieldValuesChanged = { formValues ->
-                        sheetViewModel.updateSelection(
-                            transformToPaymentSelection(
-                                formValues,
-                                selectedItem
+                        lifecycleScope.launch {
```

Comment:
I'm concerned about this change. Making this async gives us the chance to introduce race conditions. Realistically, we should store the job, and cancel the previous one.

We also can't use lifecycle scope here. Since we want the work to finish even if the fragment/activity goes away. But then we introduce a leak (so we'd need to refactor this further).